### DISCUSSION TITLE
Academy: Button should be redirecting to Crashcourse Affiliate Advanced, not to staking Advanced

### DISCUSSION THREAD

**Discussion by bf-mike-mkt** [betfinio/public#109] [Bugs and Reports]
<img width="1504" alt="image" src="https://github.com/user-attachments/assets/b1462238-509f-4d5e-a562-ba19d21fbb78" />

**Comment by bf-monster** [comment]
Hey, @bf-mike-mkt  still an issue?

**Comment by bf-mike-mkt** [comment]
yes it is not fixed yet

**Comment by bf-monster** [comment]
Should be fixed now
